src/main/scala-2.13.0-M5/scala/swing/MutableShim.scala

```diff
@@ -0,0 +1,41 @@
+/*                     __                                               *\
```

Comment:
What is the reason we have scala-2.13 versus scala-2.13.0-M5 source directories?

Reply:
That sbt issue means it doesn't pick up 2.13, but would on official version?